src/features/amUI/agentDetails/AgentDetails.tsx

```
@@ -68,6 +68,12 @@ export const AgentDetails = () => {
     clients,
   });
   const [searchString, setSearchString] = useState<string>('');
+const [expandedIds, setExpandedIds] = useState<string[]>([]);
```

ESLint

[error] Insert `··`

(prettier/prettier)